src/test/regress/expected/subquery_basics.out

```diff
@@ -503,3 +503,89 @@ DEBUG:  Plan XXX query after replacing subqueries and CTEs: SELECT user_id FROM
 ---------------------------------------------------------------------
 (0 rows)
 
+CREATE TABLE dist(id int, value int);
```

Comment:
can you load few hundred rows to the tables that the subqueries/intermediate results would return some tuples? Otherwise, I'm not sure if we'd get the errors like:

> ERROR: row field count is 2, expected 1 CONTEXT: while executing command on localhost:5432